app/views/application/_header.html.erb

```diff
@@ -1,35 +1,67 @@
+        <div class="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start">
+          <div class="flex-shrink-0">
+            <%= image_tag('UVBrigadeLogo.png', class: 'block lg:hidden h-8 w-auto', alt: 'UV Brigade logo') %>
+            <%= image_tag('UVBrigadeLogo.png', class: 'hidden lg:block h-8 w-auto', alt: 'UV Brigade logo') %>
+          </div>
+          <div class="hidden sm:block sm:ml-6">
+            <div class="flex">
+              <a href="#" class="px-3 py-2 rounded-md text-sm font-medium leading-5 text-white bg-gray-900 focus:outline-none focus:text-white focus:bg-gray-700 transition duration-150 ease-in-out">Home</a>
+              <a href="#" class="ml-4 px-3 py-2 rounded-md text-sm font-medium leading-5 text-gray-300 hover:text-white hover:bg-gray-700 focus:outline-none focus:text-white focus:bg-gray-700 transition duration-150 ease-in-out">About</a>
+              <a href="#" class="ml-4 px-3 py-2 rounded-md text-sm font-medium leading-5 text-gray-300 hover:text-white hover:bg-gray-700 focus:outline-none focus:text-white focus:bg-gray-700 transition duration-150 ease-in-out">How to use</a>
+              <a href="#" class="ml-4 px-3 py-2 rounded-md text-sm font-medium leading-5 text-gray-300 hover:text-white hover:bg-gray-700 focus:outline-none focus:text-white focus:bg-gray-700 transition duration-150 ease-in-out">Login</a>
+            </div>
+          </div>
+        </div>
+      </div>
     </div>
-    <div>
-      <button id="triggerMenu" type="button" class="block hover:text-white focus:text-white focus:outline-none">
-        <svg class="h-6 w-6 fill-current" viewBox="0 0 24 24">
-          <path id="menuIcon" fill-rule="evenodd" d="M4 5h16a1 1 0 0 1 0 2H4a1 1 0 1 1 0-2zm0 6h16a1 1 0 0 1 0 2H4a1 1 0 0 1 0-2zm0 6h16a1 1 0 0 1 0 2H4a1 1 0 0 1 0-2z" />
-        </svg>
-      </button>
+
+
+    <!--
+      Mobile menu, toggle classes based on menu state.
+
+      Menu open: "block", Menu closed: "hidden"
+    -->
+    <div id="mobile-menu" class="hidden sm:hidden">
+      <div class="px-2 pt-2 pb-3">
+        <a href="#" class="block px-3 py-2 rounded-md text-base font-medium text-white bg-gray-900 focus:outline-none focus:text-white focus:bg-gray-700 transition duration-150 ease-in-out">Home</a>
+        <a href="#" class="mt-1 block px-3 py-2 rounded-md text-base font-medium text-gray-300 hover:text-white hover:bg-gray-700 focus:outline-none focus:text-white focus:bg-gray-700 transition duration-150 ease-in-out">About</a>
+        <a href="#" class="mt-1 block px-3 py-2 rounded-md text-base font-medium text-gray-300 hover:text-white hover:bg-gray-700 focus:outline-none focus:text-white focus:bg-gray-700 transition duration-150 ease-in-out">How to use</a>
+        <a href="#" class="mt-1 block px-3 py-2 rounded-md text-base font-medium text-gray-300 hover:text-white hover:bg-gray-700 focus:outline-none focus:text-white focus:bg-gray-700 transition duration-150 ease-in-out">Login</a>
```

Comment:
Hmm it looks like maybe we lost the links to "/users/sign_in" and "/users/sign_up" and "destroy_user_session_path" with this code change?

Reply:
this is proving to be a terrible pull request! did i merge the wrong stuff?? Will investigate now....

Reply:
It's not terrible! These things happen often :)

Reply:
Look at it, i realize what I did... i added the styling before the devise stuff had been merged in ... so this branch doesn't have any of the devise account management stuff in it....the navigation header needs to be redone with the tailwind css and the devise helpers and links....but i'm not sure how to accomplish that.

Reply:
No big deal! It should be as simple as adding another commit that brings in the Devise changes needed. I would just keep this GitHub PR open and then copy over the necessary Device helpers and such and then make a new commit.

It think you had fixed the merge conflicts in the UI, which probably means you need to pull the latest changes in this remote branch to your local machine to work with. I would think you could just checkout your branch, then do a git pull to pull in the latest changes that you had committed via the GitHub UI. Then make a new commit with the Devise stuff in there and you should be good to go

Reply:
Roger. I will try that.

Reply:
ok...sorry...one more question...git is a weakness for me...

do i need to redo the devise stuff on this branch? or can i merge the devise branch into this branch? i'm confused because i have the devise stuff already merged into the master....right? so i'm just not getting how I keep from messing that up.